Demo/src/main/java/com/paypal/android/api/model/Order.kt

```diff
@@ -2,12 +2,16 @@ package com.paypal.android.api.model
 
 import android.os.Parcelable
 import kotlinx.parcelize.Parcelize
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
 
 @Parcelize
+@Serializable
 data class Order(
     val id: String? = null,
     val intent: String? = null,
     val status: String? = null,
+    @SerialName("last_digits")
     val cardLast4: String? = null,
```

Comment:
```suggestion
    val lastDigits: String? = null,
```
Same here, totally worth cleaning up to align with the actual name of the underlying resource. I think `cardLast4` as a naming convention is historically a Braintree thing.